app/reducers/imageGallery.ts

```diff
@@ -0,0 +1,32 @@
+import { produce } from 'immer';
+import { createSelector } from 'reselect';
+import createEntityReducer, { fetching } from 'app/utils/createEntityReducer';
+import { ImageGallery, File } from '../actions/ActionTypes';
+
+export default createEntityReducer({
+  key: 'imageGallery',
```

Comment:
```suggestion
  key: 'imageGalleryEntries',
```
The naming convention (which is aaalmost always followed) for entityReducer keys is the plural of the entity.